go/cmd/dolt/commands/diff.go

```diff
@@ -421,15 +422,28 @@ func diffSchemas(tableName string, sch1 schema.Schema, sch2 schema.Schema, dArgs
 func tabularSchemaDiff(tableName string, tags []uint64, diffs map[uint64]diff.SchemaDifference) errhand.VerboseError {
```

Comment:
What you have here is fine, but it substantially duplicates the logic in sql.SchemaAsCreateStmt (before your change as well). I think this is fine to check in as is, but a better strategy would probably be to use a textual diff library, and just output the diff in sql.SchemaAsCreateStmt between the old schema and the new one.

Reply:
not a lot of diff libraries for go that I found... this one (https://github.com/sergi/go-diff) is ok, but doesn't support line by line diffing afaik. Think it's probably better to leave this as is for now, then revisit updating this code to be less duplicative when I have a bit more time to focus on it.